OpenRA.Mods.Common/Traits/Conditions/ReloadAmmoPoolOnCondition.cs

```diff
@@ -0,0 +1,58 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Reloads an ammo pool.")]
+	public class ReloadAmmoPoolOnConditionInfo : ConditionalTraitInfo
```

Comment:
Just `ReloadAmmoPool` please.  We don't add `OnCondition` to any of our other conditional traits.